webapp/platform/components/src/compass-ui/components/button/button.overrides.ts

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {alpha, ComponentsProps, ComponentsVariants, emphasize, Theme} from '@mui/material';
+import {ComponentsOverrides} from '@mui/material/styles/overrides';
+
+import {blend} from '../../utils/color-utils';
+
+const componentName = 'MuiButton';
+
+declare module '@mui/material/Button' {
+    interface ButtonPropsSizeOverrides {
+        'x-small': true;
+    }
+
+    interface ButtonPropsVariantOverrides {
+        quaternary: true;
+    }
+}
+
+const defaultProps: ComponentsProps[typeof componentName] = {
+    disableElevation: true,
+    disableRipple: true,
+    disableTouchRipple: true,
+    disableFocusRipple: true,
+};
+
+const getFocusStyles = (color: string) => ({
+    '&:not(.Mui-disabled)': {
```

Comment:
we can wrap the whole style block in `` marks, I am not sure if they are applied correctly this way?